src/core/api/models/admin_job.go

```diff
@@ -59,20 +64,21 @@ type ScheduleParam struct {
 	Cron string `json:"cron"`
 }
 
-// AdminJobRep holds the response of query gc
+// AdminJobRep holds the response of query admin job
 type AdminJobRep struct {
-	ID           int64          `json:"id"`
-	Name         string         `json:"job_name"`
-	Kind         string         `json:"job_kind"`
-	Schedule     *ScheduleParam `json:"schedule"`
-	Status       string         `json:"job_status"`
-	UUID         string         `json:"-"`
-	Deleted      bool           `json:"deleted"`
-	CreationTime time.Time      `json:"creation_time"`
-	UpdateTime   time.Time      `json:"update_time"`
+	ID   int64  `json:"id"`
+	Name string `json:"job_name"`
+	Kind string `json:"job_kind"`
+	AdminJobSchedule
```

Comment:
Please see the above comment, I wish to suggest shadowed attribute be put on the top.